charts/prometheus-operator-crds/values.yaml

```diff
@@ -2,3 +2,24 @@
 ##
 crds:
   annotations: {}
+
+  alertmanagerconfigs:
+    enabled: true
+  alertmanagers:
+    enabled: true
+  podmonitors:
+    enabled: true
+  probes:
+    enabled: true
+  prometheusagents:
+    enabled: true
+  prometheuses:
+    enabled: true
+  prometheusrules:
+    enabled: true
+  scrapeconfigs:
+    enabled: true
+  servicemonitors:
+    enabled: true
+  thanosrulers:
+    enabled: true
```

Comment:
```suggestion
    enabled: true

```

Reply:
I added comments for all properties following the Helm conventions in 4448089f936904c7dcaaf98d41328a42113744e0 @QuentinBisson